From d6f9611c3759801581ca1db9608a0fcd43185360 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 24 Dec 2022 11:58:20 +0000
Subject: [PATCH] libstdc++: fix incomplete type of class Group

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of class Group for initilization of
Section..
---
 components/omnibox/browser/autocomplete_grouper_sections.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/omnibox/browser/autocomplete_grouper_sections.h b/components/omnibox/browser/autocomplete_grouper_sections.h
index b0b865a..9f5dca1 100644
--- a/components/omnibox/browser/autocomplete_grouper_sections.h
+++ b/components/omnibox/browser/autocomplete_grouper_sections.h
@@ -8,9 +8,9 @@
 #include <memory>
 #include <vector>
 
+#include "components/omnibox/browser/autocomplete_grouper_groups.h"
 #include "components/omnibox/browser/autocomplete_match.h"
 
-class Group;
 class Section;
 using PGroups = std::vector<std::unique_ptr<Group>>;
 using PSections = std::vector<std::unique_ptr<Section>>;
-- 
2.38.2

